qiita_pet/support_files/doc/source/tutorials/no-raw-sequences.rst

```diff
@@ -55,3 +58,30 @@ compression program to use is ``gzip``:
     # new files, named barcodes.fastq.gz, forward.fastq.gz and revers.fastq.gz
     gzip *.fastq
 
+
+Per sample FASTQ files without barcode and primer information
+=============================================================
+
+In this example we assume we are working with uncompressed per-sample FASTQ
+files and that they do not have any barcode or primer information. This file type
+normally is what you can download from `Illumina's BaseSpace <https://basespace.illumina.com/home/index>`__.
+
+
+The current system allows to upload this kind of files. The only requirement is
+that the prep template should have the uploaded sequence file(s) name in the
+run_prefix field. This should be an exact match without extension (fastq or
```

Comment:
I think you should mention this in the documentation about preparing templates, under the `Required fields for pre-processing target gene data` section.